app/views/posts/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>show the posts of a user</h1>
\ No newline at end of file
+<section class="show-user">
+  <h1>Show all Posts of a user</h1>
+
+  <% render 'users/user', user: @user %>
+
+   <article class="user d-flex">
+    <div class="img-container">
+      <img src="<%= @user.photo %>" alt="User_Photo">
+    </div>
+    <div class="details d-flex">
+      <h2 class="name"><%= @user.name %></h2>
+      <span class="sub">Number of posts: <%= @user.postsCounter %></span>
+    </div>
+  </article>
+
+  <%# <div class="posts">
+    <%= render  @posts %>
+  <%# </div> %> 
```

Comment:
- [ ] It looks like the posts#index page does not display posts 😢 It may be due to the **above code being commented thereby not rendering the appropriate partial template** please make sure that the pages shows all posts by a particular author/user 😃 👍